webapp/components/wpt-results.html

```diff
@@ -227,13 +233,23 @@
           this.refreshDisplayedNodes();
         };
 
+        // Add a diff testRun spoof, if needed.
```

Comment:
What's being spoofed here? Looks like it's just fetching the diff if it's needed?

A comment on the UX of this view. The diff column isn't shown at all until the data is loaded, which takes at least a second. Would it make sense to show at least the header first so that it's clear something is loading?

Reply:
The `TestRun` object doesn't exist in the datastore, so we add it here. Maybe, but adding a nice async UI is probably overkill for first cut :)